source/index.txt

```diff
@@ -79,85 +82,79 @@ MongoDB Documentation
    .. card::
       :headline: C
       :url: https://www.mongodb.com/docs/drivers/c/
-      :icon: /icons/c.svg
+      :icon: /images/icons/c.svg
       :icon-alt: C Driver icon
 
    .. card::
       :headline: C++
       :url: https://www.mongodb.com/docs/drivers/cxx/
-      :icon: /icons/c++.svg
+      :icon: /images/icons/c++.svg
       :icon-alt: C plus plus Driver icon
 
    .. card::
       :headline: C#
       :url: https://www.mongodb.com/docs/drivers/csharp/current/
-      :icon: /icons/csharp.svg
+      :icon: /images/icons/csharp.svg
       :icon-alt: C sharp Driver icon
 
    .. card::
       :headline: Go
       :url: https://www.mongodb.com/docs/drivers/go/current/
-      :icon: /icons/go.svg
+      :icon: /images/icons/go.svg
       :icon-alt: Go Driver icon
 
    .. card::
       :headline: Java
       :url: https://www.mongodb.com/docs/drivers/java-drivers/
-      :icon: /icons/java.svg
+      :icon: /images/icons/java.svg
       :icon-alt: Java Driver icon
    
-   .. card::
```

Comment:
minor note! why did we remove Kotlin?

Reply:
@seungpark good question. It wasn't in the [Figma](https://www.figma.com/file/SxKPGkhEJ8ixM9giIPEuJC/Docs-Homepage-Redesign?node-id=1351%3A59951&mode=dev) designs.

Reply:
Lets put Kotlin back, but otherwise, LGTM.